README.md

```diff
@@ -266,7 +266,10 @@ Example `Jenkinsfile` to use the opinionated infrastructure pipeline:
 
 def product = "rhubarb"
 
-withInfraPipeline(product) {
+//Optional
+def component = "extra-detail" 
```

Comment:
this should be a separate example, people tend to copy paste and given it's not our default approach then it shouldn't be included in this example

Reply:
I have added a new section. please review